src/main/java/ch/njol/skript/lang/SkriptParser.java

```diff
@@ -692,12 +692,13 @@ public final <T> Expression<? extends T> parseExpression(final Class<? extends T
 						isLiteralList &= t instanceof Literal;
 						ts.add(t);
 						if (b != 0) {
-							final String d = expr.substring(pieces.get(b - 1)[1], x).trim();
+							final String d = expr.substring(pieces.get(b - 1)[1], x).trim().toLowerCase(Locale.ENGLISH);
 							if (!d.equals(",")) {
+								boolean or = !d.contains("nor") && d.endsWith("or");
```

Comment:
Nor means negated of or. There is no support for this. If 1 is 1 nor 2 would return true

Reply:
> If 1 is 1 nor 2 would return true

It would not. [`nor` in expression lists works like `and`](https://github.com/SkriptLang/Skript/pull/5062/files#diff-7f18778d8310e3fc62cd38a0d472a6648c230a13050136890cd5d20294d967daR699)